examples/ct/handlers_test.go

```diff
@@ -163,7 +163,6 @@ func TestPostHandlersOnlyAcceptPost(t *testing.T) {
 	// Anything in the post handler list should only accept POST
 	for _, hp := range allPostHandlersForTest(client) {
 		s := httptest.NewServer(hp.handler)
-		defer s.Close()
```

Comment:
Yeah, given that it's test code, I think it's more likely that someone might mistakenly add a `continue` later and miss out on the `Close`.

(Presumably the best pattern for real code would be to put the inner body of the loop into a function of its own, but clearly overkill here.)